util/fine_tune/fine_tune_stats.py

```diff
@@ -0,0 +1,22 @@
+import typer
+import subprocess
+
+
+def fine_tune(openai_api_key: str, results_file_id: str, output_file: str):
+    # Check if required inputs are provided
+    if not (openai_api_key and results_file_id and output_file):
+        typer.echo("Error: Missing required parameters")
+        typer.echo("Usage: ./fine_tune.py fine-tune <OPENAI_API_KEY> <RESULTS_FILE_ID> <OUTPUT_FILE>")
+        raise typer.Exit(code=1)
+
+
+    curl_command = [
```

Comment:
It's better to make the request using the `requests` library or the `openai.File` object.